DeviceIdentification/Implementation/Broadcom/Broadcom.cpp

```diff
@@ -78,39 +78,63 @@ class DeviceImplementation : public PluginHost::ISubSystem::IIdentifier {
 private:
     inline void UpdateDeviceInfo(string& identifier, string& chipset, string& firmwareVersion) const
     {
-        static constexpr uint8_t MaxInfoCollection = 3;
-        static constexpr const TCHAR ChipsetKey[] = _T("Chip ID");
-        static constexpr const TCHAR IdentifierKey[] = _T("CHIPID");
-        static constexpr const TCHAR FirmwareVersionKey[] = _T("Nexus Release");
+        identifier.assign(extractSerialNumber());
+        chipset.assign(extractChipId());
+        firmwareVersion.assign(extractFirmwareVersion());
+    }
+
+    inline std::string extractSerialNumber() const
```

Comment:
Serial Number is already provided by https://rdkcentral.github.io/rdkservices/#/api/DeviceInfoPlugin?id=serialnumber. Checking why we need to duplicate.